rwengine/src/engine/GameState.hpp

```diff
@@ -301,6 +301,7 @@ class GameState {
     float cutsceneStartTime;
     /** Flag for rendering cutscene letterbox */
     bool isCinematic;
+    int hudFlash;
```

Comment:
I think we should initialize it. In order to prevent UB.

Reply:
done

Reply:
Instead of int, this could be an enum class so that the possible values are well named.